### DISCUSSION TITLE
Improve coverage and accuracy for repository (client) software information

### DISCUSSION THREAD

**Discussion by KellyStathis** [datacite/datacite-suggestions#98] [General Suggestions]
### What is the problem that your suggestion solves?

Knowing which integrations (e.g. [repository platform integrations](https://support.datacite.org/docs/service-provider-software-integrations)) are used by which DataCite users would help with:
- supporting users in troubleshooting DataCite integrations 
- sharing information with Registered Service Providers (RSPs) about how their integrations are being used by DataCite users
- analyzing metadata practices and driving metadata quality improvements through engagement with integration providers (since this mediates which metadata is sent to DataCite)
- targeting outreach for users of specific platforms 
- identifying integrations which are not yet part of the RSP program, so DataCite staff can engage with the organizations building them

### What solution might meet your needs?

We currently have an optional "software" field for repositories. This is optional and is [only populated for 432 repositorie](https://api.datacite.org/clients?query=software:*)

There are a couple challenges with this field:
-  it's not required - so we have mostly blank values and can't do much with the information for analysis
- the dropdown values for this field require dev team involvement to update; engagement can't easily do this
- there is no mechanism for repository users to clarify a custom integration/other selection

We might explore:
- making the "software" field required for new (and existing?) repositories created in Fabrica
- expanding the list of options
- allowing free text (or a supplemental field) for when the options aren't applicable
- making it easier for engagement staff to update the controlled list when we learn of new platforms
- pulling software information from re3data where applicable (part of re3data schema) - but need to be careful not to overwrite what is submitted directly to DataCite if we use this source

### Your name

Kelly Stathis

### Your organization

DataCite

### What group(s) would benefit from your suggestion?

- [ ] DataCite members registering DOIs and metadata and administering accounts
- [X] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [X] DataCite staff
- [ ] Other

**Comment by codycooperross** [comment]
Improved software coverage and wider software options would support organizations like ROR when working to drive adoption of software-supported PIDs, metadata, and features. 

[Link for DataCite staff](https://app.frontapp.com/open/cnv_badq191?key=iTWmZViTk7cH_mCIZj_kzinlZj0pzbFw)
